Tigger/bin/tigger_convert.py

```diff
@@ -517,33 +522,41 @@ def convert_coordinates(coords):
                 parser.error("Invalid --add-brick setting %s" % brickspec)
             if [src.name for src in sources if src.name == srcname]:
                 print("Error: model already contains a source named '%s'" % srcname)
-            # add brick
-            from astropy.io import fits as pyfits
-            from astLib.astWCS import WCS
 
-            input_hdu = pyfits.open(fitsfile)[0]
+            input_hdu = fits.open(fitsfile)[0]
             hdr = input_hdu.header
             max_flux = float(input_hdu.data.max())
-            wcs = WCS(hdr, mode='pyfits')
-            # Get reference pixel coordinates
-            # wcs.getCentreWCSCoords() doesn't work, as that gives us the middle of the image
-            # So scan the header to get the CRPIX values
-            ra0 = dec0 = 1
-            for iaxis in range(hdr['NAXIS']):
-                axs = str(iaxis + 1)
-                name = hdr.get('CTYPE' + axs, axs).upper()
-                if name.startswith("RA"):
-                    ra0 = hdr.get('CRPIX' + axs, 1) - 1
-                elif name.startswith("DEC"):
-                    dec0 = hdr.get('CRPIX' + axs, 1) - 1
-            # convert pixel to degrees
-            ra0, dec0 = wcs.pix2wcs(ra0, dec0)
+
+            wcs, refpix, refsky, ra_axis, dec_axis = get_wcs_info(hdr)
+            naxis = hdr["NAXIS"]
+            if ra_axis is None or dec_axis is None:
+                print("Can't find RA and/or DEC axis in this FITS image")
+                sys.exit(1)
+
+            ra0, dec0 = refsky[ra_axis], refsky[dec_axis]
+
+            # convert reference pixel from degrees to rad
             ra0 *= DEG
             dec0 *= DEG
-            sx, sy = wcs.getHalfSizeDeg()
+            # get NX, NY
+            nx, ny = input_hdu.data.shape[-1:-3:-1]
```

Comment:
Again assumes the axis will be the last axes in the header here.
Why not read this from NAXIS{ra_axis} and NAXIS{dec_axis}?